build.sbt

```diff
@@ -266,7 +266,10 @@ lazy val zincCompileCore = (project in internalPath / "zinc-compile-core")
   .settings(
     name := "zinc Compile Core",
     libraryDependencies ++= Seq(scalaCompiler.value % Test, launcherInterface, parserCombinator),
-    unmanagedJars in Test := Seq(packageSrc in compilerBridge in Compile value).classpath
```

Comment:
We can already use trailing commas here, can you add one to `sourceManaged` so that this doesn't happen again?